api/spec/v1.yaml

```diff
@@ -2562,10 +2562,12 @@ components:
           description: The available balance, in base units.
         escrow:
           allOf: [$ref: '#/components/schemas/TextBigInt']
-          description: The active escrow balance, in base units.
+          description: |
+            Effectively only useful on validator accounts: active escrow balance, in base units.
```

Comment:
Maybe 

> Amount delegated to the account, in base units. Effectively only useful on validator accounts.

and

> Debonding amount delegated to the account, in base units. Effectively only useful on validator accounts.

But what you did is fine as well.

Reply:
How about:
Amount delegated to this validator, in base units.
Amount debonding from this validator, in base units.

Reply:
👍